library/common-library/templates/_license_secret.yaml.tpl

```diff
@@ -0,0 +1,22 @@
+{{/*
+Renders the license key secret if user has not specified a custom secret.
+*/}}
+{{- define "common.license.secret" }}
+{{- if not (include "common.license._customSecretName" .) }}
+
+{{/* Fail if licenseKey is empty and required */}}
```

Comment:
As this `if` has no `-}}` the comment should be like this:
```mustache
{{- if not (include "common.license._customSecretName" .) }}

{{- /* Fail if licenseKey is empty and required */}}
```
Although I prefer this:

```suggestion
{{- if not (include "common.license._customSecretName" .) -}}

{{/* Fail if licenseKey is empty and required */}}
```

Reply:
This is a nit. It only renders empty lines.

Reply:
I think we will get an empty line here anyway as comments cannot have line chompers :(

Reply:
[They can](https://github.com/newrelic/helm-charts/blob/master/library/common-library/templates/_low-data-mode.tpl#L10-L14)

Reply:
Last time I tried this I got a syntax error... TIL.